src/frontend/utils/chat/chat.ts

```diff
@@ -0,0 +1,31 @@
+import { converse } from 'utils/window';
+
+export const generateAnonymousNickname = (): string => {
```

Comment:
```suggestion
export const generateAnonymousNickname = () => {
```

TS can infers the type here.

Reply:
done in [94260d](https://github.com/openfun/marsha/pull/1276/commits/94260daa50c03f34070086bde57fb2de9a43b5f8)